patches/minecraft/net/minecraft/command/impl/TeleportCommand.java.patch

```diff
@@ -1,5 +1,17 @@
 --- a/net/minecraft/command/impl/TeleportCommand.java
 +++ b/net/minecraft/command/impl/TeleportCommand.java
+@@ -121,6 +_,11 @@
+    }
+ 
+    private static void func_201127_a(CommandSource p_201127_0_, Entity p_201127_1_, ServerWorld p_201127_2_, double p_201127_3_, double p_201127_5_, double p_201127_7_, Set<SPlayerPositionLookPacket.Flags> p_201127_9_, float p_201127_10_, float p_201127_11_, @Nullable TeleportCommand.Facing p_201127_12_) throws CommandSyntaxException {
++      net.minecraft.util.math.vector.Vector3d target = net.minecraftforge.event.ForgeEventFactory.onEntityTeleportCommand(p_201127_1_, p_201127_3_, p_201127_5_, p_201127_7_);
++      if (target == null) return;
++      p_201127_3_ = target.field_72450_a;
++      p_201127_5_ = target.field_72448_b;
++      p_201127_7_ = target.field_72449_c;
+       BlockPos blockpos = new BlockPos(p_201127_3_, p_201127_5_, p_201127_7_);
+       if (!World.func_234935_k_(blockpos)) {
+          throw field_241077_a_.create();
```

Comment:
Checking the position is loaded? Explain why this is added to the patch.

Reply:
I didn't add this, so I'm not sure how it appeared.

Reply:
Actually, this is automatically generated to show the relevant lines around the patch. This is automatic. Notice how there is no `+` or `-` at the start of the line, meaning this was not changed and is just being displayed.

Reply:
Ah yes, a patch of a patch. Truly the bane of our collective existence.